### ISSUE TITLE
Deprecated exporter_locator always imported

### ISSUE THREAD

**Issue by SylvainCorlay** [jupyter/nbconvert#1033]
exporter_locator is always imported in `export.py` while it is deprecated.

```
./export.py:from .exporter_locator import *
./exporter_locator.py:warnings.warn("""`nbconvert.exporters.exporter_locator` is deprecated in favor of `nbconvert.exporters.base` since nbconvert 5.0.""",
```

**Comment by MSeal** [comment]
Let's remove it for 6.0. I think we'll have another patch release or two in 5.5.x. Perhaps we should start a 6.0 branch with this as the first change?

**Comment by t-makaro** [comment]
Closed by #1230
